### PR TITLE
dir: add page

### PR DESCRIPTION
**Pull request by itsjzt**
- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
  `windows/`
- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

**Comment by sbrl** [comment]
Hey, thanks for the new page, @itsjzt! Looks like you're having a bit of trouble with the build, though. We've documented the format we prefer [here](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#markdown-format), and a few good example pages are ones like `tar`, and `7z`.

Also, if you wouldn't mind, could you sign the contributor license agreement (CLA) by clicking the link that @CLAassistant has posted above? You'll need to do so before we can merge your new page.

Thanks again!

**Comment by itsjzt** [comment]
just  signed  contributor license agreement (CLA)  and fixed the error using tldr-lint, now it can be merged 😸

File: pages/windows/dir.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# DIR
```
Comment (agnivade) [inline pages/windows/dir.md]: lower case please.

File: pages/windows/dir.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# DIR 
+
+> Displays file and folders in current directory or given directory.
```
Comment (agnivade) [inline pages/windows/dir.md]: How about "List directory contents" ? This matches with the description of `ls`

File: pages/windows/dir.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+- Show list of files and folder of current directory:
+
+`dir`
+
+- Show list of file and folder in any other directory:
+
+`dir {{path/to/<placeholder>}}`
```
Comment (agnivade) [inline pages/windows/dir.md]: The placeholder is meant to be replaced with some value. Please change it to path/to/directory

File: pages/windows/dir.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# DIR 
+
+> Displays file and folders in current directory or given directory.
+
+- Show list of files and folder of current directory:
```
Comment (agnivade) [inline pages/windows/dir.md]: folder => folders

of current directory => "in" current directory

File: pages/windows/dir.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+`dir`
+
+- Show list of file and folder in any other directory:
```
Comment (agnivade) [inline pages/windows/dir.md]: in any other directory => in a given directory

file => files

folder => folders

File: pages/windows/dir.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+`dir {{path/to/<placeholder>}}`
+
+- Show hidden file and folders in current directory:
```
Comment (agnivade) [inline pages/windows/dir.md]: Same comments as above. Please change files and folders uniformly to plural. And update the placeholder text.

File: pages/windows/dir.md (hunk 2 above)
Comment (itsjzt) [reply pages/windows/dir.md]: yeah, sure

File: pages/windows/dir.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+- Show contents of current directory:
+
+`dir`
+
+- Show contents of a given directory:
+
+`dir {{path/to/directory}}`
+
+- Show hidden contents of current directory:
```
Comment (agnivade) [inline pages/windows/dir.md]: Just saying "Show hidden contents" might give the impression that it shows _only_ hidden contents. How about "Show contents of current directory, including hidden files" ?

File: pages/windows/dir.md [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+
+`dir /A:H`
+
+- Show hidden contents of any other directory:
```
Comment (agnivade) [inline pages/windows/dir.md]: Same as above. Also please change "any other" to "a given"

File: pages/windows/dir.md [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+
+`dir  {{path/to/directory}} /A:H`
```
Comment (agnivade) [inline pages/windows/dir.md]: remove the double space

File: pages/windows/dir.md (hunk 7 above)
Comment (itsjzt) [reply pages/windows/dir.md]: It only shows the hidden content

Comment (agnivade) [reply pages/windows/dir.md]: In that case, just do /A. Because I think generally people will always expect to see the normal files.

Comment (itsjzt) [reply pages/windows/dir.md]: done.

**Comment by sbrl** [comment]
@itsjzt Just made a few minor grammatical changes. Are they ok?

**Comment by agnivade** [comment]
Hey @itsjzt - this PR is waiting for you to acknowledge that the changes made by @sbrl are okay and you agree with them. Please let us know your thoughts.

**Comment by itsjzt** [comment]
Yeah that great, thanks @sbrl

**Pull request merged**